randovania/games/dread/game_data.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 from randovania.games import game
+from randovania.games.dread.gui.dread_teleporter_details_tab import DreadTeleporterDetailsTab
```

Comment:
this import needs to be inside `_gui`

Reply:
actually just import in `dread/gui/__init__.py` and use `gui.DreadTeleporterDetailsTab`

Reply:
Like this?
Can you tell me why it has to be this way? I don't really understand the reason atm.

Reply:
we support headless installation and operation with a limited feature set, so anything that depends on qt must only be imported when running in gui mode

Reply:
Most importantly, the _server_. It's the test that failed even!